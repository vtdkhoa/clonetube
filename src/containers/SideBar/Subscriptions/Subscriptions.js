import React, { Component, Fragment } from 'react'
import Subscription from './Subscription/Subscription'
import SideBarHeader from '../SideBarHeader/SideBarHeader'
import { Divider } from 'semantic-ui-react'

class Subscriptions extends Component {
  render() {
    return (
      <Fragment>
        <SideBarHeader title="Subscriptions"/>
        <Subscription label="Music Channel" broadcasting/>
        <Subscription label="Coursea" amountNewVideos={10}/>
        <Subscription label="TEDx Talks" amountNewVideos={23}/>
        <Subscription label="Stanford iOS" amountNewVideos={4}/>
        <Subscription label="Udacity" amountNewVideos={114}/>
        <Divider/>
      </Fragment>
    )
  }
}

export default Subscriptions