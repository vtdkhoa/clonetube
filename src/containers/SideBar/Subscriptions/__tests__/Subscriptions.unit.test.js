import React from 'react'
import Subscriptions from '../Subscriptions'
import { shallow } from 'enzyme'

describe('Subscriptions', () => {
  test('renders', () => {
    const wrapper = shallow(<Subscriptions/>)
    expect(wrapper).toMatchSnapshot()
  })
})