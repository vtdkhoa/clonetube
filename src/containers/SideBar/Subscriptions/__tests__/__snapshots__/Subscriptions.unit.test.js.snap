// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subscriptions renders 1`] = `
<Fragment>
  <SideBarHeader
    title="Subscriptions"
  />
  <Subscription
    broadcasting={true}
    label="Music Channel"
  />
  <Subscription
    amountNewVideos={10}
    label="Academind"
  />
  <Subscription
    amountNewVideos={23}
    label="Gearbox Digital"
  />
  <Subscription
    amountNewVideos={4}
    label="Udemy"
  />
  <Subscription
    amountNewVideos={114}
    label="Udacity"
  />
  <Divider />
</Fragment>
`;
